### ISSUE TITLE
Read additional info (IOB, CAGE, Units left in reservoir, ... )

### ISSUE THREAD

**Issue by nielsmaerten** [nielsmaerten/nightscout-assistant#56]
**Is your feature request related to a problem? Please describe.**
Currently, Nightscout Status only says my BG and then quits. I'd like to be able
to ask for other parameters as well. For example Insulin on Board (IOB), Cannula age (CAGE), ...

**Describe the solution you'd like**
We could add more Intents. Currently there's only 'Glucose Status', wich reads out the last reading. 
We could add an 'IOB' intent, 'COB' intent, and so on.

_OR_: We could let the user select which parameters they'd like to hear on the site, and 
then read out the chosen parameters from the same single Glucose Status intent.

Would like some feedback on what you think is preferable :)

**Comment by many7695** [comment]
For me, would be to let us select the parameters we want to hear on the site

**Comment by ErwynLein** [comment]
Yeah, a choice for what we want to hear is great. Maybe multiple command for multiple choice ? If I want my BG, I don't want my IOB, but maybe I will want it later... But not the BG. Choice it is, for me !

**Comment by many7695** [comment]
What is the current status of adding new parameters

**Comment by nielsmaerten** [comment]
Sorry it's taking so long.
I've designed the feature but also just started a new job, so I need to find some time to actually write the code. I'll keep updating this issue with any progress.

**Comment by many7695** [comment]
Is there any new information if or when this might be implemented. I appreciate all the work you have done with this and your glucose ticker

**Comment by nielsmaerten** [comment]
@many7695 It's still being worked on. Haven't forgotten about it yet :)
Last couple months have been crazy though (oh hi, COVID-19) so I can only put in a few hours during the weekend.

**Comment by danradigan** [comment]
Are use the google home integration and love it! Thank you so much for your work. I have added the integration to my good night routine so that I know I’m going to bed every night safely.

What I’d love to hear: glucose value, trend, insulin on board, carb on board and how recent the data is.

I’m wondering if it would be easier to just have short (glucose value, trend, recent) medium (the items above) and long potentially reading out everything you could get access to

**Comment by nielsmaerten** [comment]
Hi @ErwynLein @danradigan 

Good news! These features are now available in the successor to Nightscout Assistant:  
[**Gluco Check**](https://github.com/nielsmaerten/gluco-check) 🎉

Gluco Check is still in development, but our first bèta testers have started using it.  
If you'd like to try it out, shoot me an [email](mailto:hello@niels.me) or a message on [Gitter](https://gitter.im/nielsmaerten)
